### ISSUE TITLE
Suggestion: Improvement in Profile Page

### ISSUE THREAD

**Issue by Mushkir** [anburocky3/cyberdude-internship-profiles#4]
Interns' LinkedIn profile integration in Profiles page. In case in the future any recruiters/others want to explore all the interns' LinkedIn profiles, it will be helpful to them...

**Comment by ghost** [comment]
@Mushkir yes 👍 , but we can provide the social links too! isn't it great?

**Comment by anburocky3** [comment]
> @Mushkir yes 👍 , but we can provide the social links too! isn't it great?

Definitely yes. If you guys are so curious, you can also do some UI changes and can give a pull request.

**Comment by Mushkir** [comment]
@anburocky3 Yes sure, I will do the demo UI asap
